prow/OWNERS

```diff
@@ -1,4 +1,6 @@
 # See the OWNERS docs at https://go.k8s.io/owners
+options:
+  no_parent_owners: true
```

Comment:
Does this mean that oncall can't add quick fixes if approvers are on vacation or something?

Reply:
Yes. If on-call has to make hotfixes to the product code instead of reverting deployment artifacts we're in a really bad place. The approvers will do our due diligence to make sure there is someone around.